### PR TITLE
Declare this version of Cloud SQL Proxy deprecated.

### PR DESCRIPTION
**Pull request by nikkithurmond**

### CODE REVIEW

File: cloud_sql_proxy/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+This version of Cloud SQL Proxy has been deprecated. Please use the [official Cloud SQL Proxy](https://github.com/GoogleCloudPlatform/cloudsql-proxy).
+
+
+## Questions?
```
Comment (yanweiguo) [inline cloud_sql_proxy/README.md]: Can we remove `Questions` and `Licensing` sections since they are covered in root readme?

**Comment by yanweiguo** [comment]
Can we also mark fluentd deprecated?

File: cloud_sql_proxy/README.md (hunk 1 above)
Comment (nikkithurmond) [reply cloud_sql_proxy/README.md]: For sure, let me do that.

**Comment by nikkithurmond** [comment]
I accidentally pushed a deprecation README for fluentd without getting it reviewed (still new to Github), but @yanweiguo gave it an LGTM, so I'll leave it.

**Pull request merged**